index.js

```diff
@@ -2,42 +2,66 @@
 var argv = process.argv;
 
 module.exports = (function () {
+	var support = function (level) {
+		if (level <= 0) return null;
+		var result = {};
+		result.level = level;
+		result.has16 = result.hasBasic = level >= 1;
+		result.has256 = level >= 2;
+		result.has16m = level >= 3;
+		return result;
+	}
+
 	if ('FORCE_COLOR' in process.env) {
-		return true;
+		return support(1);
```

Comment:
`FORCE_COLOR` currently means *has basic support*. All modules that have depended on *supports-color* have assumed this to mean 16 colors.

There are four levels in the PR:

0. no color support
1. 16 color support
2. 256 color support
3. 16 million color support